spec/factories/experimentation/reminder_templates.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :reminder_template, class: Experimentation::ReminderTemplate do
+    message { "Your appointment is in three days" }
+    appointment_offset { -3 }
```

Comment:
Lets not force negative numbers here...we could use `offset.days.from_now` in the code, and I think we can avoid having negatives like this.

Reply:
Not sure I follow. The offset is relative to the appointment, not relative to now.